src/browser-frontend/views/browser/chrome/tabbar/tab.css

```diff
@@ -47,6 +48,32 @@ specific language governing permissions and limitations under the License.
   height: var(--theme-tab-open-height);
 }
 
+.tab.has-owner {
+  left: 0;
+  transition:
+    width 0.05s var(--animation-curve),
+    background-position 0.2s var(--animation-curve),
+    height 0.2s var(--animation-curve),
+    left 0.2s var(--animation-curve);
```

Comment:
Also: if you're going to break these on different lines (which I'm liking a lot), you might as well update the other `transition` rules as well?